server/testutils/factories/prisonAddress.ts

```diff
@@ -0,0 +1,19 @@
+import { faker } from '@faker-js/faker'
```

Comment:
We can use `@faker-js/faker/locale/en_GB` to get proper UK postcodes (and more UK-style data in general). Can do the same in `prison`, but the post code is where it most noticeably affects the format

Reply:
Good spot, have added this and hardcoded 'United Kingdom' as it didn't make sense for things like "Uganda" to show with a UK postcode. We can revisit this in future if we need but I don't think we'll need to go more granular in our test data just yet.